pkg/istio/virtualservice_test.go

```diff
@@ -152,6 +155,12 @@ var _ = Describe("Operations for istio VirtualService kind", func() {
 
 					Expect(GetMutatedRoute(mutatedVirtualService, targetV1Host, targetV1Subset).Redirect).To(BeNil())
 				})
+				It("include destinations with no subset", func() {
```

Comment:
I just wonder if it would bring some benefit to cover more boundary cases as the conditionals grew a bit. WDYT?

Reply:
explain?